spec/tasks/bundle-audit/targets/finding_1/Gemfile.lock

```diff
@@ -0,0 +1,13 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    kafo (0.3.1)
+
+PLATFORMS
+  ruby
+
+RUBY VERSION
+   ruby 2.3.3
```

Comment:
We should have a project discussion about ruby versions ... I think this would effectively also bump the Ruby version ... which in theory is a good thing but just something we should probably stop and think about...

Reply:
For this particular case the ruby version specification is not required since this `Gemfile.lock` is used only as a test vector, it should not have impact on the ruby version being used by the system.
This being said, and to make it more explicit, I can remove the references to ruby versions from the test data files referenced in this PR since they don't provide value in this case. What do you think?

P.S. This does not remove the (obviously) very good discussion you propose of bumping the Ruby version but probably that would be in a different scope.

Reply:
Got it.  Thanks.  I'm approving this PR and merging.

I think we can probably bump the ruby version easily.  I would like to see @omerlh's input too.